csv_importer.go

```diff
@@ -0,0 +1,251 @@
+package uadmin
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"strings"
+)
+
+// CSVImporterHandler handles CSV files
+func CSVImporterHandler(w http.ResponseWriter, r *http.Request, session *Session) {
+	if err := r.ParseMultipartForm(32 << 20); err != nil {
```

Comment:
pls add comments/description why to use `32 << 20` - sounds like magic

Reply:
It's 32 Mb. I just use the same approach I see everywhere in uadmin...